kubernetes/loculus/values.schema.json

```diff
@@ -1034,6 +1034,9 @@
       "properties": {
         "ncbiGatewayUrl": {
           "type": ["string", "null"]
+        },
+        "mirrorBucket": {
+          "type": ["string", "null"]
```

Comment:
Maybe add a comment that _any_ non-empty string enables the feature, even `"False"` does :)